File: BackEnd/backend/src/main/java/com/fitapp/backend/infrastructure/persistence/entity/enums/PackageType.java
package com.fitapp.backend.infrastructure.persistence.entity.enums;

public enum PackageType {
    SPORT_PACK,      // deportes + sus parámetros predefinidos + categorías
    PARAMETER_PACK,  // solo parámetros custom
    ROUTINE_PACK,    // rutinas completas (incluye ejercicios si son custom)
    EXERCISE_PACK,   // ejercicios custom con sus parámetros
    MIXED            // combinación de varios tipos
}

File: FrontEnd/app/src/main/java/com/fitapp/appfit/feature/marketplace/model/enums/PackageType.kt
package com.fitapp.appfit.feature.marketplace.model.enums

enum class PackageType(val displayName: String) {
    SPORT_PACK("Pack de Deporte"),
    PARAMETER_PACK("Pack de Parámetros"),
    ROUTINE_PACK("Pack de Rutinas"),
    EXERCISE_PACK("Pack de Ejercicios"),
    MIXED("Pack Mixto");

    companion object {
        fun fromString(value: String?): PackageType? {
            return try {
                valueOf(value?.uppercase() ?: return null)
            } catch (e: IllegalArgumentException) {
                null
            }
        }
    }
}